spring-cloud-dataflow-server-kubernetes-dependencies/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.2.1.RELEASE</version>
+		<relativePath />
+	</parent>
+	<artifactId>spring-cloud-dataflow-server-kubernetes-dependencies</artifactId>
+	<version>1.1.0.BUILD-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<name>spring-cloud-dataflow-server-kubernetes-dependencies</name>
+	<description>Spring Cloud Data Flow Server for Kunernetes Dependencies BOM designed to support consumption of Spring Cloud Data Flow Server for Kubernetes from the Spring Initializr.</description>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dataflow-server-kubernetes</artifactId>
```

Comment:
^^ the only point I see to having a single-dependency bom, is to have the ability to add [additional-boms](https://github.com/ericbottard/initializr/blob/AddDataFlowK8S/initializr-service/src/main/resources/application.yml#L51) to initializr.

But this seems more like a fundamental limitation of initializr (a checkbox there needs to map to only one dep) than a feature here.